src/components/localisation/routes.ts

```diff
@@ -168,7 +169,9 @@ const checkLanguageFormat = (language: LanguageOption) => {
     language.description &&
     typeof language.description === 'string' &&
     language.flag &&
-    typeof language.flag === 'string'
+    typeof language.flag === 'string' &&
+    language.locale &&
+    typeof language.locale === 'string'
```

Comment:
```suggestion
    (language.locale ? typeof language.locale === 'string' : true)
```

Allows it to be optional but still check type if present.